integration_tests/src/test/kotlin/integration/MultipleDevicesIT.kt

```diff
@@ -0,0 +1,32 @@
+package integration
+
+import FlankCommand
+import com.google.common.truth.Truth.assertThat
+import run
+import org.junit.Test
+
+class MultipleDevicesIT {
+    private val name = this::class.java.simpleName
+
+    @Test
+    fun `run tests on multiple devices - android`() {
+        val name = "$name-android"
+        val result = FlankCommand(
+            flankPath = "../test_runner/build/libs/flank.jar",
```

Comment:
we use 
`../test_runner/build/libs/flank.jar`
as path many times, let's move it to some shared `val`

Reply:
Good point, thanks